src/devices/Pcf8563/Pcf8563.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Pcf8563
+{
+    /// <summary>
+    /// Realtime Clock PCF8563
+    /// </summary>
+    public class Pcf8563 : IDisposable
+    {
+        private I2cDevice _i2cDevice = null;
+
+        /// <summary>
+        /// PCF8563 Default I2C Address
+        /// </summary>
+        public const byte DefaultI2cAddress = 0x51;
+
+        /// <summary>
+        /// DS3231 DateTime
+        /// </summary>
+        public DateTime DateTime { get => ReadTime(); set => SetTime(value); }
+
+        /// <summary>
+        /// Creates a new instance of the PCF8563
+        /// </summary>
+        /// <param name="i2cDevice">The <c>System.Device.I2c.I2cDevice</c></param>
+        public Pcf8563(I2cDevice i2cDevice)
+        {
+            _i2cDevice = i2cDevice;
+
+            // Set "Normal Mode"
+            Span<byte> ctrl1Config = stackalloc byte[] { (byte)Register.PCF_CTRL1_ADDR, 0x00 };
```

Comment:
Is there a way for this binding to check if the deviceID of the device matches what we expect? Other I2c devices have a device id that we can check in initialization just to make sure that the device we are talking to is indeed a PCF8563

Reply:
I looked through the datasheet carefully and found no registers like DeviceID

Reply:
ok, I guess that should be fine then. I wonder if cases like this we should add a try catch on the first read/write and send a helpful error message in case of having the wrong device, or perhaps connecting it to the board incorrectly. You don't have to add this in your PR, this is just food for thought :)